lib/engine/game/base.rb

```diff
@@ -90,6 +90,9 @@ class Base
       EBUY_PRES_SWAP = true # allow presidential swaps of other corps when ebuying
       EBUY_OTHER_VALUE = true # allow ebuying other corp trains for up to face
 
+      # 1882: When laying a token that is from another corp, the other corp takes ownership
+      LAY_TOKEN_OTHER_CORP_TAKES_OWNERSHIP = false
```

Comment:
again this is way too specific, we should move this ONLY to the 1882 class

Reply:
Yep, I wasn't sure the best way to express it :)